GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -24,15 +26,20 @@ public FormOpenDirectory(GitModule currentModule)
             InitializeComponent();
             Translate();
 
-            _NO_TRANSLATE_Directory.DataSource = GetDirectories(currentModule);
-
-            Load.Select();
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                await TaskScheduler.Default.SwitchTo(alwaysYield: true);
```

Comment:
:bulb: Explicit `SwitchTo` is unnecessary (caller is on the UI thread)

:bulb: No need to switch to a background thread here since the code is calling straight into asynchronous code anyway.